.travis.yml

```diff
@@ -1,14 +1,18 @@
 language: python
 python:
     - "2.7"
-    - "2.6"
+    - "3.4"
+
+matrix:
+  allow_failures:
+    - python: "3.4"
```

Comment:
I assume this is temporary config for adapting python3.4 porting

Reply:
@chunfuwen exactly!